time time_stamp 
uint8 status
float64[7] state_trueVec
float64[7] state_measuredVec
float64[2] input_actVec
float64[2] input_desVec
